kube-hunter/kube-hunter-plugin.yaml

```diff
@@ -0,0 +1,16 @@
+sonobuoy-config:
+  driver: Job
+  plugin-name: kube-hunter
+  result-format: raw
+spec:
+  command:
+  - /bin/sh
+  - -c
+  - "python kube-hunter.py --pod --report=json | tee /tmp/results/report.json && echo -n /tmp/results/report.json > /tmp/results/done"
+  image: sonobuoy/kube-hunter:v0
```

Comment:
Which version/commit did you use for this image? I think it would be helpful if we could include that info in the image tag. I see their initial/only release had the tag "v0.2.0". Could we use that here?

Reply:
💯 Yes let me do that. I'll rebuild, tag and push.